scripts/src/main/resources/scripts/command/dotnet

```diff
@@ -23,7 +23,7 @@ function doSetup() {
     if doIsMacOs || doIsWindows
     then
       doDownload "${download_url}"
-      doRunCommand "~/Downloads/${fileSetup}"
+      doRunCommand "$HOME/Downloads/${fileSetup}"
```

Comment:
has to be `${DEVON_HOME_DIR}` instead.
See here:
https://github.com/devonfw/ide/blob/master/documentation/variables.asciidoc